docs/website/src/pages/index.tsx

```diff
@@ -71,22 +71,22 @@ export default function Home(): JSX.Element {
                   Push code fast and often
                 </h2>
                 <p className="padding-horiz--md">
-    Spend less time maintaining your deployment infrastructure and more time coding. Immediately have your application running each time you compile.
+    Spend less time maintaining your development environment and more time coding. Immediately have your application up-to-date each time you modify the sources.
                 </p>
               </div>
             </div>
           </div>
         </div>
         <div className={styles.overview}>
           <div className="container text--center margin-top--md">
-            <h1>A developer-focused tool for blazingly fast Kubernetes application development</h1>
+            <h1>A developer-focused tool for blazingly fast Container-based application development</h1>
             <div className="row">
               <div className="col col--6">
                 <h2 className={clsx(styles.featureHeading)}>
                   Initialize and develop your application
                 </h2>
                 <p className="padding-horiz--md">
-                  Only two commands away from deploying on a cluster! Use <code>odo</code> to initialize and then develop your application directly on the cluster.
+                  Only two commands away from developing on a cluster! Use <code>odo</code> to initialize and then develop your application directly on the cluster.
```

Comment:
Maybe we can also remove mentions of "cluster" altogether?

```suggestion
                  Only two commands away from developing in a container! Use <code>odo</code> to initialize and then develop your application directly on your container platform of choice.
```

PS: there is another occurrence here: https://github.com/redhat-developer/odo/pull/6988/files#diff-4d059b04386b14da6c2ebf32cf1433b63a33e7ff3f8cae9523f87894d253dbfbR108